### PR TITLE
Remove the underline of vineyard operator arch and add a FAQ for vineyard operator

### PR DESCRIPTION
**Pull request by dashanji**
What do these changes do?
-------------------------

* Remove the underline of vineyard operator arch.
* Add a FAQ for vineyard operator.
* Delete the default field of CRD in the vineyard operator doc.

Related issue number
--------------------

Fixes https://github.com/v6d-io/v6d/issues/1219

### CODE REVIEW

**Comment by sighingnow** [comment]
Put FAQ contents to https://github.com/v6d-io/v6d/blob/main/docs/notes/developers/faq.rst

**Comment by dashanji** [comment]
> https://github.com/v6d-io/v6d/blob/main/docs/notes/developers/faq.rst

Okay.

File: docs/notes/developers/faq.rst [hunk 1]
```diff
@@ -88,6 +88,43 @@ concerns, please feel free to `open an issue`_ or `post it to discussions`_.
...
+11. *How to connect to the vineyard cluster deployed by the vineyard operator?*
+
+  There are two ways to connect to the vineyard cluster deployed by the vineyard operator:
+  - `Through IPC`. Create a pod with the specific labels so that the pod can be scheduled 
+    to the node where the vineyard cluster is deployed.
+  - `Through RPC`. Connect to the vineyard cluster through the RPC service exposed by the 
+    vineyard operator. You could refer to the `guide`_ for more details.
+
+  Is there a way to install the vineyard cluster on Kubernetes quickly?
+  ----------------------------------------------------------------------
```
Comment (sighingnow) [inline docs/notes/developers/faq.rst:121]: Why a title line here? Can you use **xxx** for bold instead?

Comment (dashanji) [reply docs/notes/developers/faq.rst]: It's my bad. I forgot to update it. Thanks for catching this and addressed.

File: docs/notes/developers/faq.rst [hunk 2]
```diff
@@ -88,6 +88,42 @@ concerns, please feel free to `open an issue`_ or `post it to discussions`_.
...
+12. *Is there a way to install the vineyard cluster on Kubernetes quickly?*
+
+  To reduce the complexity of the installation, we provide a `command line tool`_
+  to install the vineyard cluster on Kubernetes quickly.
 
 .. _open an issue: https://github.com/v6d-io/v6d/issues/new
 .. _post it to discussions: https://github.com/v6d-io/v6d/discussions/new
+.. _cert-manager: https://cert-manager.io/
+.. _guide: ../../tutorials/kubernetes/using-vineyard-operator.rst
+.. _command line tool: ../../../k8s/cmd/README.md
```
Comment (sighingnow) [inline docs/notes/developers/faq.rst:129]: Don't refer to the readme, refer to our documentation page in documentations.

**Pull request merged**